app/services/project_data_service.rb

```diff
@@ -0,0 +1,90 @@
+require "base64"
+
+class ProjectDataService
+
+  def initialize(project, access_token)
+    @project = project
+    @access_token = access_token
+    @client = Octokit::Client.new(access_token: access_token)
+    @circle_config = circle_config
+  end
+
+  def github_org
+    Organization.find(@project.organization_id).name
+  end
+
+  def project_name
+    url = git_remote.split("/").last
+    url.gsub(".git", "")
+  end
+
+  def git_remote
+    stage = @project.stages&.detect{ |s| s.name == "master" }
+    stage && stage.git_remote
+  end
+
+  def computed_properties
+    {
+      base_libraries: {
+        ruby: ruby_version,
+        node: node_version
+      },
```

Comment:
These are more "runtimes" than libraries. Rails and express and so on would be libraries though.

Reply:
If you wanted to keep this general (runtimes, libraries, etc.) you might just call them "dependencies."